### PR TITLE
Add mechanize recipe

### PR DESCRIPTION
**Pull request by synapticarbors**

### CODE REVIEW

**Comment by synapticarbors** [comment]
All of the automated tests pass on this recipe, but the conda-forge-linter fails. The recipe is based on previous recipes I've used that have linted without issue. The only different is the `skip` under `build` since this is python 2 only. 

Is this a bug in the linter? Or am I missing something?

/cc @conda-forge/core

File: recipes/mechanize/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+test:
+  imports:
+    - mechanize
+
+about:
+  home: http://wwwsearch.sourceforge.net/mechanize/
+  license: BSD License or Zope Public License
+  summary: 'Stateful programmatic web browsing.'
+  description: |
+    Stateful programmatic web browsing in Python, after Andy Lester’s Perl module WWW::Mechanize.
```
Comment (jakirkham) [inline recipes/mechanize/meta.yaml]: It is probably struggling with this unicode character, `’`. Please change to `'`.

File: recipes/mechanize/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - mechanize
+
+about:
+  home: http://wwwsearch.sourceforge.net/mechanize/
+  license: BSD License or Zope Public License
```
Comment (jakirkham) [inline recipes/mechanize/meta.yaml]: Which BSD license? Please add clauses e.g. `BSD 3-Clause`. Also please add the version of `Zope Public License` and please feel free to shorten to `ZPL`.

Comment (jakirkham) [inline recipes/mechanize/meta.yaml]: Is there a license file we can include? If not, can we please ask them to include one. Please xref this back here.

Comment (synapticarbors) [inline recipes/mechanize/meta.yaml]: Would the following count as a valid license file?: https://github.com/jjlee/mechanize/blob/master/COPYING.txt

Comment (jakirkham) [inline recipes/mechanize/meta.yaml]: Yep, that would work great. Please add `license_file: COPYING.txt`.

**Comment by jakirkham** [comment]
Thanks @synapticarbors.

**Pull request merged**